assets/javascripts/discourse/controllers/endorse-user.js

```diff
@@ -0,0 +1,8 @@
+import Controller from "@ember/controller";
+import ModalFunctionality from "discourse/mixins/modal-functionality";
+
+export default Controller.extend(ModalFunctionality, {
+  onShow() {
+    this.set("afterSave", () => this.send("closeModal")).bind(this)
```

Comment:
What's the `bind` for here? Seems unnecessary.

Reply:
If I try to pass it without bind, `this` changes to the component where it is called, and `this.send("closeModal")` errors. This way I bind the `this` to the controller.